elfi/bo/acquisition.py

```diff
@@ -95,23 +97,55 @@ class LCBSC(AcquisitionBase):
     would be t**(2d + 2).
     """
     
-    def __init__(self, *args, delta=.1, **kwargs):
+    def __init__(self, *args, delta=.1, noise_cov=0., **kwargs):
         super(LCBSC, self).__init__(*args, **kwargs)
         if delta <= 0 or delta >= 1:
             raise ValueError('Parameter delta must be in the interval (0,1)')
         self.delta = delta
+        if isinstance(noise_cov, float) or isinstance(noise_cov, int):
```

Comment:
Oh, good, didn't know that.

I suppose we should generally add more argument checking for user-facing routines.